### PR TITLE
Destrucutes orb

### LINKED ISSUE #2: Destructure orb
Destructure the orb to more easily add components in the future.
https://circleci.com/docs/2.0/local-cli/#packing-a-config

This can also be accomplished with the Orb Starter Kit: 
https://github.com/CircleCI-Public/orb-starter-kit

### PR DESCRIPTION
**Pull request by beatbrot**
This fixes #2

### CODE REVIEW

**Comment by beatbrot** [comment]
Sure, will do :)

File: examples/standard-swift-project.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  as CircleCI artifacts and test results
+usage:
+  version: 2.1
+
+  orbs:
+    swift: circleci/swift@1.0.0
+
+  workflows:
+    build-and-test:
+      jobss:
```
Comment (lex111) [inline examples/standard-swift-project.yml]: Typo :)

```suggestion
      jobs:
```

Comment (beatbrot) [reply examples/standard-swift-project.yml]: Ooops :D

**Comment by KyleTryon** [comment]
Looks great to me 👍  Thank you very much @lex111 and @beatbrot

**Pull request merged**